.changeset/wild-mails-work.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-pantheon-decoupled-kit": minor
+---
+
+Related content ACF component addon for next-wp
```

Comment:
Sorry, missed this the first time:
Think about how this will look on the changelog when we do a release. In this case I think it could be formatted a bit differently so that it tells a reader not familiar with ACF or an addon what this feature does.
```suggestion
Added a new addon, `next-wp-acf-addon`. It includes an example implementation of data sourced from WordPress with the [Advanced Custom Fields plugin](https://www.advancedcustomfields.com/) activated.
```